backend/backend/is_bankrupt.py

```diff
@@ -0,0 +1,48 @@
+"""Module for providing the functionality to
+   detirmine if a player is declared bankrupt
+"""
+
+import json
+import sys
+import cgitb
+from backend.player import Player
+from backend.game import get_games
+from backend.properties import Property, get_properties
+
+cgitb.enable()
+
+
+def is_bankrupt(source=sys.stdin, output=sys.stdout):
+    """Player is checked if they are bankrupt or not"""
+
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request['player_id']
+
+    with Player(player_id) as player:
```

Comment:
Since you’re just reading a property of the player, I don’t think you need the `with` context manager here. Does it run ok using it here and also inside `player_remove()`?

Reply:
I fixed that and also removed having to use 'with Player' in 'player_remove'.

Reply:
Do you not need to use it in `player_remove`?